docs/index.md

```diff
@@ -52,6 +52,12 @@ import ReactModal from 'react-modal';
   /* Function that will be run when the modal is requested
      to be closed (either by clicking on overlay or pressing ESC).
      Note: It is not called if isOpen is changed by other means. */}
+     
+  onOverlayRightClick={
```

Comment:
I would propose `onOverlayContextMenu` as a better name for this prop, per @diasbruno's suggestion [here](https://github.com/reactjs/react-modal/pull/961#issuecomment-1222428392).

We are passing this to React's `onContextMenu` event handler, which maps to the `contextmenu` event (see [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event)):

> The contextmenu event fires when the user attempts to open a context menu. This event is typically triggered by clicking the right mouse button, or by pressing the context menu key.

Some alternate ways to trigger the `contextmenu` event beyond right click:
- `ctrl` + click on MacOS 
- `shift+f10` on Windows
- `fn+12` on MacOS (with "Alternate Pointer Events" enabled)
 
...And some screen readers have their own triggers, too.

Reply:
Note that if we do decide to rename the prop, it may be worth updating the commits/PR title so that the wording in the changelog is accurate.

Reply:
> fn+12 on MacOS (with "Alternate Pointer Events" enabled)

This one I didn't know. :)

Reply:
Yep, agree. Better to keep the names consistent.